][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_eventLogHandler::Diagnostic_eventLogHandler::writeEventToLog(s)] __end__ Total: 1 milliseconds.
[ 2852 4892][17 Oct 20:08:35][TR_CONN_MANAGER] diagnostic_reporter: entering...
[ 2852 4892][17 Oct 20:08:35][TR_CONN_MANAGER] TR_CONN_MANAGER::CreateAndSendNotification: TrEventManager is not ready yet - ignore this notification - return
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __end__ Total: 1 milliseconds.
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __end__ Total: 1 milliseconds.
[ 2852 4892][17 Oct 20:08:35][TracService] Software\CheckPoint\TRAC\5.0 exists - no need to backup
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::create: __start__ 20:8:35.639
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::ConfigurationManager: __start__ 20:8:35.639
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::init(): __start__ 20:8:35.639
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::load(): __start__ 20:8:35.639
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::loadDefaultParams()
[ 2852 4892][17 Oct 20:08:35][FileManager] FileManager::openFile: file "trac.defaults" does not exist
[ 2852 4892][17 Oct 20:08:35][FileManager] FileManager::readFile: error occured while opening the file "trac.defaults"
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::loadDefaultParams() error reading trac.defaults file in ConfigurationManager::loadDefaultParams()
[ 2852 4892][17 Oct 20:08:35][RaisException] RaisException::BaseException::BaseException: error code = 11, error message = error reading file in ConfigurationManager::loadDefaultParams(), file name =ConfigurationManager.cpp, line number = 1369
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::load(): __end__ 20:8:35.639. Total time - 0 milliseconds
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::init(): __end__ 20:8:35.639. Total time - 0 milliseconds
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::ConfigurationManager: __end__ 20:8:35.639. Total time - 0 milliseconds
[ 2852 4892][17 Oct 20:08:35][CONFIG_MANAGER] ConfigurationManager::create: __end__ 20:8:35.639. Total time - 0 milliseconds
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __start__
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [DEBUG] [Diagnostic_EventReporter::report_helpdesk_event_1(s)]  eventDisplayType: 4 severityLevel: 1 userName:  siteName:  gwName:  gwIP:  eventText: Error in client initialization
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __start__
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [INFO] [Diagnostic_Id_Generator::generateId(s)] new id is: timeStamp: 1666055315, serialNumWithinTheSecond: 1
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __end__ Total: 0 milliseconds.
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __start__
[ 2852 4892][17 Oct 20:08:35][XmlWrapper] XmlDocument::XmlDocument: inside
[ 2852 4892][17 Oct 20:08:35][XmlWrapper] XmlDocument::SaveString: inside
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [DEBUG] [Diagnostic_EventReporter::_report_event(s)] new event after removing 0d0a: 

<OBJECT name="helpdesk event" type="DiagnosticHelpdeskEventData">

  <OBJECT name="idD" type="Diagnostic_RecordId">
    <Field name="timeStampD" value="1666055315"></Field>
    <Field name="serialD" value="1"></Field>
  </OBJECT>

  <Field name="dataTypeD" value="1"></Field>

  <Field name="dataDisplayTypeD" value="4"></Field>

  <Field name="userNameD" value=""></Field>

  <Field name="siteNameD" value=""></Field>

  <Field name="gwNameD" value=""></Field>

  <Field name="gwIPD" value=""></Field>

  <Field name="severityLevelD" value="1"></Field>

  <Field name="dataD" value="Error in client initialization"></Field>

  <Field name="helpdeskRecordTypeD" value="-1"></Field>

</OBJECT>

[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_eventLogHandler::Diagnostic_eventLogHandler::writeEventToLog(s)] __start__
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_eventLogHandler::Diagnostic_eventLogHandler::writeEventToLog(s)] __end__ Total: 0 milliseconds.
[ 2852 4892][17 Oct 20:08:35][TR_CONN_MANAGER] diagnostic_reporter: entering...
[ 2852 4892][17 Oct 20:08:35][TR_CONN_MANAGER] TR_CONN_MANAGER::CreateAndSendNotification: TrEventManager is not ready yet - ignore this notification - return
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __end__ Total: 1 milliseconds.
[ 2852 4892][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __end__ Total: 1 milliseconds.
[ 2852 4892][17 Oct 20:08:35][TR_OVERSITE] TR_OVERSITE::Create: EXCEPTION!
[ 2852 4892][17 Oct 20:08:35][TR_OVERSITE] TR_OVERSITE::Create: error reading file in ConfigurationManager::loadDefaultParams()
[ 2852 4892][17 Oct 20:08:35][TracService] ServiceMain: mainloop exited without stopping the service, calling exit
[ 2852 4892][17 Oct 20:08:35][] COM_Initializer::Dtor: nothing to do
[ 2852 4892][17 Oct 20:08:35][proxy_API] ~OS_API
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::~MultiSiteAutoUpdateManager] __start__
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::reset] __start__
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::setAutoUpdateState] __start__
[ 2852 4892][17 Oct 20:08:35][Multi-site] [INFO] [MultiSiteAutoUpdateManager::setAutoUpdateState]  moving form state eAutoUpdateState_preInit to state eAutoUpdateState_preInit
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::setAutoUpdateState] __end__ Total: 0 milliseconds.
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::reset] __end__ Total: 0 milliseconds.
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteAutoUpdateManager::~MultiSiteAutoUpdateManager] __end__ Total: 0 milliseconds.
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteEncDomainMapper::~MultiSiteEncDomainMapper] __start__
[ 2852 4892][17 Oct 20:08:35][Multi-site] [COVERAGE] [MultiSiteEncDomainMapper::~MultiSiteEncDomainMapper] __end__ Total: 0 milliseconds.
[ 10588 12240][17 Oct 20:08:35][TracService] ServiceMain: Entered service's main entry point (Check Point Endpoint Security)
[ 10588 12240][17 Oct 20:08:35][cpservice] service_report_status_to_scm: Reporting start pending
[ 10588 12240][17 Oct 20:08:35][TracService] IsDisabledInEps: failed to read mode from registry
[ 10588 12240][17 Oct 20:08:35][TracService] 

[ 10588 12240][17 Oct 20:08:35][TracService] -------------------------------------------------------------
[ 10588 12240][17 Oct 20:08:35][TracService] 					SERVICE WAS STARTED 						
[ 10588 12240][17 Oct 20:08:35][TracService] -------------------------------------------------------------

[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __start__
[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [DEBUG] [Diagnostic_EventReporter::report_helpdesk_event_1(s)]  eventDisplayType: 0 severityLevel: 1 userName:  siteName:  gwName:  gwIP:  eventText: Service was started
[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __start__
[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [INFO] [Diagnostic_Id_Generator::generateId(s)] new id is: timeStamp: 1666055315, serialNumWithinTheSecond: 0
[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __end__ Total: 0 milliseconds.
[ 10588 12240][17 Oct 20:08:35][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __start__
[ 10588 12240][urationManager::loadDefaultParams()
[ 12564 6964][17 Oct 20:02:09][FileManager] FileManager::openFile: file "trac.defaults" does not exist
[ 12564 6964][17 Oct 20:02:09][FileManager] FileManager::readFile: error occured while opening the file "trac.defaults"
[ 12564 6964][17 Oct 20:02:09][CONFIG_MANAGER] ConfigurationManager::loadDefaultParams() error reading trac.defaults file in ConfigurationManager::loadDefaultParams()
[ 12564 6964][17 Oct 20:02:09][RaisException] RaisException::BaseException::BaseException: error code = 11, error message = error reading file in ConfigurationManager::loadDefaultParams(), file name =ConfigurationManager.cpp, line number = 1369
[ 12564 6964][17 Oct 20:02:09][CONFIG_MANAGER] ConfigurationManager::load(): __end__ 20:2:9.102. Total time - 1 milliseconds
[ 12564 6964][17 Oct 20:02:09][CONFIG_MANAGER] ConfigurationManager::init(): __end__ 20:2:9.102. Total time - 1 milliseconds
[ 12564 6964][17 Oct 20:02:09][CONFIG_MANAGER] ConfigurationManager::ConfigurationManager: __end__ 20:2:9.102. Total time - 1 milliseconds
[ 12564 6964][17 Oct 20:02:09][CONFIG_MANAGER] ConfigurationManager::create: __end__ 20:2:9.102. Total time - 1 milliseconds
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __start__
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [DEBUG] [Diagnostic_EventReporter::report_helpdesk_event_1(s)]  eventDisplayType: 4 severityLevel: 1 userName:  siteName:  gwName:  gwIP:  eventText: Error in client initialization
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __start__
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [INFO] [Diagnostic_Id_Generator::generateId(s)] new id is: timeStamp: 1666054929, serialNumWithinTheSecond: 1
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_Id_Generator::generateId(s)] __end__ Total: 0 milliseconds.
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __start__
[ 12564 6964][17 Oct 20:02:09][XmlWrapper] XmlDocument::XmlDocument: inside
[ 12564 6964][17 Oct 20:02:09][XmlWrapper] XmlDocument::SaveString: inside
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [DEBUG] [Diagnostic_EventReporter::_report_event(s)] new event after removing 0d0a: 

<OBJECT name="helpdesk event" type="DiagnosticHelpdeskEventData">

  <OBJECT name="idD" type="Diagnostic_RecordId">
    <Field name="timeStampD" value="1666054929"></Field>
    <Field name="serialD" value="1"></Field>
  </OBJECT>

  <Field name="dataTypeD" value="1"></Field>

  <Field name="dataDisplayTypeD" value="4"></Field>

  <Field name="userNameD" value=""></Field>

  <Field name="siteNameD" value=""></Field>

  <Field name="gwNameD" value=""></Field>

  <Field name="gwIPD" value=""></Field>

  <Field name="severityLevelD" value="1"></Field>

  <Field name="dataD" value="Error in client initialization"></Field>

  <Field name="helpdeskRecordTypeD" value="-1"></Field>

</OBJECT>

[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_eventLogHandler::Diagnostic_eventLogHandler::writeEventToLog(s)] __start__
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_eventLogHandler::Diagnostic_eventLogHandler::writeEventToLog(s)] __end__ Total: 0 milliseconds.
[ 12564 6964][17 Oct 20:02:09][TR_CONN_MANAGER] diagnostic_reporter: entering...
[ 12564 6964][17 Oct 20:02:09][TR_CONN_MANAGER] TR_CONN_MANAGER::CreateAndSendNotification: TrEventManager is not ready yet - ignore this notification - return
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::_report_event(s)] __end__ Total: 0 milliseconds.
[ 12564 6964][17 Oct 20:02:09][DIAGNOSTIC] [COVERAGE] [Diagnostic_EventReporter::report_helpdesk_event_1(s)] __end__ Total: 0 milliseconds.
[ 12564 6964][17 Oct 20:02:09][TR_OVERSITE] TR_OVERSITE::Create: EXCEPTION!
[ 12564 6964][17